docs/docs/guide/express-js.mdx

```diff
@@ -0,0 +1,168 @@
+});
+
+// Initialize it
+client.init().then(() => {
+  console.log("GrowthBook initialized");
+});
+```
+
+Lets also add a middleware to `index.js` that creates a user-scoped instance for every request. Make sure to place this above the route handlers:
+
+```js
+app.use((req, res, next) => {
+  // Attributes about the current user/request
+  const userContext = {
+    attributes: {
+      // In a real app this would come from a cookie or session
+      // We would also add more attributes like country, etc
+      id: "123",
+    }
+  }
+
+  // Make this available to all subsequent route handlers
+  req.growthbook = client.createScopedInstance(userContext);
+  next();
+});
+```
+
+Lets re-start the Node process and make sure you see the "GrowthBook initialized" message in the console. Since we are using a `.env` file, we need to modify the command to tell Node.js to load it.
+
+```bash
+node --env-file=.env index.js
+```
+
+## 4. Create a Feature in GrowthBook
+
+Back in the GrowthBook application, we can create a new feature. For this tutorial, we'll make a simple on/off feature flag that determines whether or not we show the message in Spanish.
+
+<img
+  src="/images/guides/express-js-1-create-feature.png"
+  alt="GrowthBook Create Feature"
+  style={{ width: 500, margin: "0 auto" }}
+/>
+
+The key we chose (spanish-greeting) is what we will reference when using the GrowthBook SDK.
+
+We can now edit the route in `index.js` to use this flag:
+
+```js
+app.get('/', (req, res) => {
+  let message = "Hello World!";
+
+  if (req.growthbook.isOn("spanish-greeting")) {
+    message = "Hola Mundo!";
+  }
+
+  res.send(message);
+})
+```
+
+Now, if you restart the Node process and refresh the page in the browser, you will still see the original "Hello World!" because when we created the feature, we set it to be Off by default.
+
+## 5. Target Specific Users
+
+Now we can add rules to the feature to turn it on for specific users.
+
+In the `userContext` we added, we hard-coded an id of "123". We can now use this id to create a rule in GrowthBook to turn on the feature for this user.
+
+On the feature page, click the "Add Rule" button and select "Force Value" as the rule type. Then add targeting by attribute and save the rule. It should look something like this:
+
+<img
+  src="/images/guides/express-js-2-targeting-rule.png"
+  alt="GrowthBook Targeting Rule"
+  style={{ width: 500, margin: "0 auto" }}
+/>
+
+Rules start out in a draft state in GrowthBook. You need to publish the draft to make it live.
+
+After publishing, restart the Node process once more and refresh the page. You should now see the message in Spanish!
+
+Try changing the id in `userContext` to something else like `456`, restart Node, and the response will now switch back to English!
+
+## Conclusion and Next Steps
+
+We showed here how to use a simple feature flag in an Express.js app and target individual users. This barely scratches the surface of what you can do with GrowthBook.
```

Comment:
I think this paragraph could be clearer:

"In this tutorial, you learned how to use a simple feature flag in an Express app and target individual users. But this barely scratches the surface of what you can do with GrowthBook."